tests/integration/security/filebased_tls_origination/destination_rule_tls_test.go

```diff
@@ -46,7 +46,7 @@ func mustReadFile(t *testing.T, f string) string {
 func TestDestinationRuleTls(t *testing.T) {
 	framework.
 		NewTest(t).
-		Features("security.egress.mtls").
+		Features("security.egress.tls.filebased").
```

Comment:
It looks like we're changing a label on an existing test.  Is this because the test was mislabeled?

Reply:
Correct. It's a tls test of filebased type.